cmd/soroban-cli/src/commands/tx/edit.rs

```diff
@@ -11,6 +11,9 @@ use stellar_xdr::curr;
 
 use crate::{commands::global, print::Print};
 
+const SCHEMA_URL: &str =
+    "https://github.com/stellar/rs-stellar-xdr/raw/main/xdr-json/curr/TransactionEnvelope.json";
```

Comment:
Should it be pinned? It can be changed between protocol versions

Reply:
Not right now. I'll update this to the tag once rs-stellar-xdr starts publishing it.

Reply:
We can backport the JSON to a minor version of rs-stellar-xdr if that's helpful. v23 won't be out for a bit.

Reply:
Maybe we can release a minor version of rs-stellar-xdr, just to move forward and not messing with previous releases.

Reply:
I think it's fine to merge this as is. This command is flagged as experimental. And then if we can get a minor rs-stellar-xdr release out, we can open a PR and update the URL and do a quick release.